package.json

```diff
@@ -23,8 +23,7 @@
   "author": "Jesse Gibson",
   "license": "MIT",
   "dependencies": {
-    "babel-runtime": "^6.11.6",
-    "gun": "^0.7.0"
```

Comment:
@PsychoLlama : Cool. I've added that back in to `devDependencies` and `peerDependencies`.